src/app/openmrs-api/order-resource.service.ts

```diff
@@ -39,20 +42,55 @@ export class OrderResourceService {
   }
 
   public getOrdersByPatientUuid(patientUuid: string, cached: boolean = false, v: string = null):
-  Observable<any> {
+    Observable<any> {
+
+      this.v = 'full';
```

Comment:
Are you sure we need to get the full payload vs getting specific data on the call?